Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/TypeConstraints.dfy.expect

```diff
@@ -110,6 +112,14 @@ module _System {
     function RotateRight(w: nat): selftype
   }
 
+  type set<+T> { }
+
+  type iset<*T> { }
+
+  type seq<+T> { }
+
+  type multiset<+T> { }
+
   type map<+T, +U> {
     var Keys: set<T>  // immutable
```

Comment:
Immutable with var? Shouldn't it be const?

Reply:
Good call. I changed the printing code to output these `SpecialField`s as `const`.